src/xds/metrics.rs

```diff
@@ -45,8 +54,26 @@ impl Metrics {
             connection_terminations.clone(),
         );
 
+        let message_count = Family::default();
+
+        registry.register(
+            "message_count",
+            "Total number of messages received (unstable)",
+            message_count.clone(),
+        );
+
+        let total_messages_size = Family::default();
+
+        registry.register(
+            "total_messages_size",
```

Comment:
See https://prometheus.io/docs/practices/naming/#metric-names for general guidance.

I think we need to make this:
```
        registry.register_with_unit(
            "xds_message",
            "Total number of bytes received (unstable)",
            Unit::Bytes,
            total_messages_size.clone(),
        );
```
. This will automatically become "istio_xds_message_bytes_total" - the istio_ prefix and bytes_total suffix are automatically added

Reply:
I think it will add the units, but not `total`. This is what I see when I curl the metrics endpoint

```
# HELP istio_xds_connection_terminations The total number of completed connections to xds server (unstable).
# TYPE istio_xds_connection_terminations counter
# HELP istio_xds_message Total number of messages received (unstable).
# TYPE istio_xds_message counter
istio_xds_message_total{url="type.googleapis.com/istio.workload.Address"} 1
istio_xds_message_total{url="type.googleapis.com/istio.security.Authorization"} 1
# HELP istio_xds_message_bytes Total number of bytes received (unstable).
# TYPE istio_xds_message_bytes counter
# UNIT istio_xds_message_bytes bytes
istio_xds_message_bytes_total{url="type.googleapis.com/istio.workload.Address"} 2527
istio_xds_message_bytes_total{url="type.googleapis.com/istio.security.Authorization"} 0

```

Reply:
# UNIT istio_xds_message_bytes bytes
istio_xds_message_bytes_**total**{url="type.googleapis.com/istio.workload.Address"} 2527

That is the part that matters. Its mismatched from the HELP since this is OpenMetrics spec (which is a bit weird...)